packages/user_packages/iotcc_mqtt.py

```diff
@@ -67,17 +67,20 @@ def run(self, registry):
         # Acquire credentials and required certificates from the registry
         identity = Identity(root_ca_cert=registry.get("broker_root_ca_cert"), username=registry.get("broker_username"),
                             password=registry.get("broker_password"),
-                            cert_file=registry.get("edge_system_cert_file"), key_file=registry.get("edge_system_key_file"))
+                            cert_file=registry.get("edge_system_cert_file"),
+                            key_file=registry.get("edge_system_key_file"))
 
         # Encapsulate TLS parameters
         tls_conf = TLSConf(cert_required="CERT_REQUIRED", tls_version="PROTOCOL_TLSv1_2", cipher=None)
 
         # Initialize DCC object with MQTT transport
-        mqtt_msg_attr = MqttMessagingAttributes(pub_topic="liota/"+registry.get("broker_username")+"/request",sub_topic="liota/"+registry.get("broker_username")+"/response")
+        mqtt_msg_attr = MqttMessagingAttributes(pub_topic="liota/" + registry.get("broker_username") + "/request",
```

Comment:
Shouldn't this be part of liota-core, instead of a user package? I thought we have default topics for iotcc.

Reply:
I think we need to discuss this as Liota already has unique topic generation based on uuid which was used in previous iotcc usecase.